youtube_dl/extractor/nest.py

```diff
@@ -0,0 +1,39 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class NestIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?video.nest\.com/clip/(?P<id>)(.mp4)?'
+    _TEST = {
+        'url': 'https://video.nest.com/clip/73ddb6bd57c4485597a76e154a4429ea.mp4',
+        'md5': '7ab4eb6d4c2480be1740cc014a76ee96',
+        'info_dict': {
+            'id': '73ddb6bd57c4485597a76e154a4429ea',
+            'ext': 'mp4',
+            'title': "\"\"",
```

Comment:
To match other changes:
```suggestion
            'title': r're:^NestCam video \w+',
```